src/actions/castAction.ts

```diff
@@ -58,61 +58,69 @@ module Plywood {
   export class CastAction extends Action {
     static fromJS(parameters: ActionJS): CastAction {
       var value = Action.jsToValue(parameters);
-      value.castType = parameters.castType;
+
+      var { outputType } = parameters;
+
+      // Back compat
+      if (!outputType && hasOwnProperty(parameters, 'castType')) {
+        outputType = (parameters as any).castType;
+      }
```

Comment:
they way I prefer to do this is:

```
var outputType = parameters.outputType || (parameters as any).castType;
```

This way we can also remove `castType` from the ActionJS (and ActionValue) interfaces